composer.json

```diff
@@ -31,7 +31,9 @@
     "require-dev": {
         "malukenho/docheader": "^0.1.5",
         "phpunit/phpunit": "^7.0.2",
-        "zendframework/zend-coding-standard": "~1.0.0"
+        "squizlabs/php_codesniffer": "^3.3.2",
+        "webimpress/coding-standard": "dev-master",
```

Comment:
Once https://github.com/webimpress/coding-standard/issues/3 has a stable release, these lines can be removed.